.github/workflows/tests.yml

```diff
@@ -18,62 +18,50 @@ on:
 jobs:
   tests:
```

Comment:
This is not just tests, this is server unit tests. I do like to see the same job for client unit tests (although there are none yet, some can easily add let's say `client/pkg/client/client_test.go` in the future).